addons/website/views/snippets/s_text_cover.xml

```diff
@@ -2,15 +2,19 @@
 <odoo>
 
 <template id="s_text_cover" name="Text Cover">
-    <section class="s_text_cover">
+    <section class="s_text_cover o_cc o_cc2">
         <div class="container-fluid">
-            <div class="row">
-                <div class="o_colored_level o_cc o_cc4 col-lg-5 pt160 pb160 px-5">
+            <div class="row o_grid_mode" data-row-count="12">
+                <div class="o_grid_item g-height-10 g-col-lg-7 col-lg-7 o_colored_level o_cc o_cc1" style="z-index: 1; grid-area: 2 / 3 / 11 / 8; --grid-item-padding-x: 24px; --grid-item-padding-y: 24px;">
                     <h1 class="display-3">Sell Online. <br/>Easily.</h1>
-                    <p class="lead">Write one or two paragraphs describing your product, services or a specific feature. To be successful your content needs to be useful to your readers.</p>
-                    <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                    <br/>
+                    <p class="lead">Sell online easily with a user-friendly platform that streamlines all the steps, including setup, inventory management, and payment processing.</p>
+                    <br/>
+                    <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-height-12 g-col-lg-6 col-lg-6 o_not_editable o_colored_level o_cc o_cc1 pt160 pb160 oe_img_bg d-none d-md-block" style="grid-area: 1 / 7 / 12 / -1; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
```

Comment:
You should double-check the grid properties here. After dropping the snippet, the **very first time** I try to resize the "card" it goes left even if I'm dragging right 😉

https://github.com/user-attachments/assets/c8ad3d12-995f-4062-b694-5bcf96c149ce